src/utils/validators.js

```diff
@@ -72,3 +72,8 @@ export function isValidCityName(cityName: string) {
   const re = /^([a-zA-Z\u0080-\u024F]+(?:. |-| |'))*[a-zA-Z\u0080-\u024F]*$/u;
   return re.test(cityName);
 }
+
+export function isValidUKPhone(phone: string) {
+  const re = /^(\+44\d{10}\b)/;
```

Comment:
Shall we support numbers those start with 0 instead of +44? In the UK it's a very common format

Reply:
we still need verification that it's UK number, so probably we need country picker, but this is for next release. As it needs to be changed, I would leave it with +44 for now, had a quick chat with Dmirty and he is ok.

Reply:
👍